pkg/kudoctl/packages/install/install.go

```diff
@@ -0,0 +1,144 @@
+// Package install provides function to install package resources
+// on a Kubernetes cluster.
+package install
+
+import (
+	"strings"
+	"time"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+type Options struct {
+	skipInstance    bool
+	wait            *time.Duration
+	createNamespace bool
+}
+
+type Option func(*Options)
+
+// SkipInstance installs only Operator and OperatorVersion
+// of an operator package.
+func SkipInstance() Option {
+	return func(o *Options) {
+		o.skipInstance = true
+	}
+}
+
+// WaitForInstance waits an amount of time for the instance
+// to complete installation.
+func WaitForInstance(duration time.Duration) Option {
+	return func(o *Options) {
+		o.wait = &duration
+	}
+}
+
+// CreateNamespace creates the specified namespace before installation.
+// If available, a namespace manifest in the operator package is
+// rendered using the installation parameters.
+func CreateNamespace() Option {
+	return func(o *Options) {
+		o.createNamespace = true
+	}
+}
+
+// Package installs an operator package with parameters into a namespace.
+// Instance name, namespace and operator parameters are applied to the
+// operator package resources. These rendered resources are then created
+// on the Kubernetes cluster.
+func Package(
+	client *kudo.Client,
+	instanceName string,
+	namespace string,
+	resources packages.Resources,
+	parameters map[string]string,
+	opts ...Option) error {
+	clog.V(3).Printf("operator name: %v", resources.Operator.Name)
```

Comment:
You probably copied it from the old method, but this logging could be friendlier. E.g. KUDO manager logs this smth. like:
```suggestion
	clog.V(3).Printf("Preparing %s/%s:%s for installation", namespace, resources.Operator.Name, resources.OperatorVersion.Spec.Version)
```
This way we also adopt logging instances/operators with namespaces (which is important for debugging)

Reply:
Good point, I copied these logs. Happy to improve them here and in the other places. Great suggestions!